dlk/python/dlk/core/operators.py

```diff
@@ -612,20 +612,27 @@ def __init__(self,
                  data: np.ndarray,
                  dimension_format: str = 'NHWC',
                  packed: bool = False,
-                 actual_shape: List[int] = []) -> None:
+                 actual_shape: List[int] = [],
+                 transposed_data=None) -> None:
```

Comment:
Please include the `type` for consistency.

Reply:
Thank you for review. Modified this.